igor-web/src/main/java/com/netflix/spinnaker/igor/codebuild/AwsCodeBuildMapper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Amazon.com, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.codebuild;
+
+import com.amazonaws.services.codebuild.model.StartBuildRequest;
+import java.util.Map;
+
+public class AwsCodeBuildMapper {
+  private static final String PROJECT_NAME = "projectName";
+
+  public static StartBuildRequest toStartBuildRequest(Map<String, Object> requestBody) {
+    String projectName = (String) requestBody.get(PROJECT_NAME);
+    return new StartBuildRequest().withProjectName(projectName);
```

Comment:
When this is a stage in a pipeline, is it common to get any other attributes like the commit ID that needs to be built?  I'm surprised not to see sourceVersion as an input

Reply:
I will add more advanced use case such as source version and environment variables in later commits. This commit is more focusing on successfully trigger a build from Spinnaker and monitor build status.